conftest.py

```diff
@@ -28,15 +27,10 @@ def inventory_page(create_driver):
 
 
 @pytest.fixture()
-def open_burger_menu(create_driver):
+def burger_menu(create_driver):
     return BurgerMenu(create_driver)
```

Comment:
What is it? Why do you need this code?

Reply:
Without this strings the code does not work.